cadc-vos-server/src/main/java/org/opencadc/vospace/server/pkg/VospacePackageRunner.java

```diff
@@ -142,11 +146,51 @@ public void setAppName(String appName) {
             this.vospaceAuthorizer = new VOSpaceAuthorizer(nodePersistence);
             this.pathResolver = new PathResolver(nodePersistence, vospaceAuthorizer);
             this.resourceID = nodePersistence.getResourceID();
+            this.baseNodeUri = getBaseNodeUri(resourceID);
         } catch (NamingException e) {
             throw new RuntimeException("BUG: NodePersistence implementation not found with JNDI key " + jndiKey, e);
         }
     }
 
+    /**
+     *
+     * @param node
+     * @param target
+     * @param relativePath
+     * @return
+     */
+    public static boolean isTargetInPackage(URI node, URI target, String relativePath) {
```

Comment:
no reason for this to be public, just package access for test is fine
don't need javadoc, especially one that doesn't say anything useful, but a plain comment with explanation would be nice; I can't tell what the args mean

I am surprised that you need 3 args for this... and that there is not a LinkNode in the arg list

Reply:
I think I already had the node and target URI at one time, seemed proper to use them as args, but that was the first pass again. Does make more sense to pass in the LinkNode. Updated.

Reply:
Forgot to complete the javadoc, updated.